app/src/main/kotlin/org/eclipse/kuksa/companion/feature/home/view/sheet/AdaptiveLine.kt

```diff
@@ -0,0 +1,110 @@
+/*
+ * Copyright (c) 2023 Contributors to the Eclipse Foundation
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ */
+
+package org.eclipse.kuksa.companion.feature.home.view.sheet
+
+import androidx.compose.foundation.background
+import androidx.compose.foundation.layout.Box
+import androidx.compose.foundation.layout.Spacer
+import androidx.compose.foundation.layout.height
+import androidx.compose.foundation.layout.width
+import androidx.compose.material3.windowsizeclass.WindowSizeClass
+import androidx.compose.material3.windowsizeclass.WindowWidthSizeClass
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.platform.LocalConfiguration
+import androidx.compose.ui.tooling.preview.Preview
+import androidx.compose.ui.unit.Dp
+import androidx.compose.ui.unit.dp
+import org.eclipse.kuksa.companion.PREVIEW_HEIGHT_DP
+import org.eclipse.kuksa.companion.PREVIEW_WIDTH_DP
+import org.eclipse.kuksa.companion.extension.getWindowSizeClass
+
+/**
+ * AdaptiveLine draws a line depending on the [WindowWidthSizeClass] of the device. If the device has a
+ * [WindowWidthSizeClass] of [WindowWidthSizeClass.Compact] it will draw a horizontal line, otherwise it will draw a
+ * horizontal line.
+ */
+@Composable
+fun AdaptiveLine(
```

Comment:
I think this class is much to specific and bound to the Sheet to be used outside of the Sheet Context and therefore would keep it there for the time being. 

However, if we change our Sheet Behavior it might be that this class will be removed anyway. So let's wait and see for now.

Reply:
AdaptiveLine was removed